<launch>
    <arg name="odom_topic" default="/base_pose_ground_truth"/>
    <arg name="go_signal" default="/prius/mode"/>
    <arg name="car_width" default="1.7526"/>
    <arg name="car_length" default="2.1"/>
    <arg name="tolerance" default="8"/> 
    <arg name="visualize" default="False"/> 

    <node pkg="ros-driving-track-evaluator" type="evaluator.py" name="evaluator_node" required="true" output="screen">
        <param name="odom_topic" value="$(arg odom_topic)"/>
        <param name="go_signal" value="$(arg go_signal)"/>
        <param name="car_width" value="$(arg car_width)"/>
        <param name="car_length" value="$(arg car_length)"/>
        <param name="tolerance" value="$(arg tolerance)"/>
        <param name="visualize" value="$(arg visualize)"/>
        <param name="world_path" value="$(find av)/worlds/custom.world"/>
    </node>
</launch>
